test/integration/patients/patient/dashboard.js

```diff
@@ -42,72 +51,86 @@ context('patient dashboard page', function() {
-        form: { data: { id: '1' } },
-        files: { data: [{ id: '1' }] },
+        owner: getRelationship(teamCoordinator),
+        state: getRelationship(stateTodo),
+        form: getRelationship(testForm),
+        files: getRelationship([{ id: '1' }], 'files'),
       },
-    };
+    });
 
     cy
       .routesForPatientAction()
       .routePatient(fx => {
-        fx.data.id = '1';
-        fx.data.relationships.workspaces.data = [
-          {
-            id: workspaceOne.id,
-            type: 'workspaces',
+        fx.data = getPatient({
+          relationships: {
+            workspaces: getRelationship(workspaceOne),
           },
-        ];
+        });
+
         return fx;
       })
       .routePatientActions(fx => {
-        fx.data = _.sample(fx.data, 3);
-
-        fx.data[0] = actionData;
-
-        fx.data[2].attributes.name = 'Third In List';
-        fx.data[2].relationships.state = { data: { id: '33333' } };
-        fx.data[2].attributes.updated_at = testTsSubtract(2);
-
-        fx.data[1].attributes.name = 'Not In List';
-        fx.data[1].relationships.state = { data: { id: '55555' } };
-        fx.data[1].attributes.updated_at = testTsSubtract(5);
+        fx.data = [
+          testActionData,
+          getAction({
+            attributes: {
+              name: 'Third In List',
+              updated_at: testTsSubtract(2),
+            },
+            relationships: {
+              state: getRelationship(stateInProgress),
+            },
+          }),
+          getAction({
+            attributes: {
+              name: 'Not In List',
+              updated_at: testTsSubtract(5),
+            },
+            relationships: {
+              state: getRelationship(stateDone),
+            },
+          }),
+        ];
 
         return fx;
       })
       .routePatientFlows(fx => {
-        fx.data = _.sample(fx.data, 3);
-
-        fx.data[0].attributes.name = 'Second In List';
-        fx.data[0].relationships.state = { data: { id: '33333' } };
-        fx.data[0].attributes.updated_at = testTsSubtract(1);
-
-        fx.data[2].attributes.name = 'Last In List';
-        fx.data[2].id = '2';
-        fx.data[2].relationships.state = { data: { id: '33333' } };
-        fx.data[2].relationships.owner = {
-          data: {
-            id: '11111',
-            type: 'teams',
-          },
-        };
-        fx.data[2].attributes.updated_at = testTsSubtract(5);
-
-        fx.data[1].attributes.name = 'Not In List';
-        fx.data[1].relationships.state = { data: { id: '55555' } };
-        fx.data[1].attributes.updated_at = testTsSubtract(5);
+        fx.data = [
+          getFlow({
+            attributes: {
+              name: 'Second In List',
+              updated_at: testTsSubtract(1),
+            },
+            relationships: {
+              state: getRelationship(stateInProgress),
+            },
+          }),
+          getFlow({
+            id: '2',
```

Comment:
similarly would be great to `getFlow` above this into some sort of `testFlow` for `testFlow.id`

Reply:
Updated 👍